src/rkopenmdao/butcher_tableau.py

```diff
@@ -61,16 +90,32 @@ def p(self):
     def p(self, p):
         self._p = p
```

Comment:
It's called `self._p` here, but `self.p` in the `__init__`. It should be the same in both cases.

Reply:
https://stackoverflow.com/questions/27395978/refer-to-a-property-from-inside-the-class-itself

Reply:
Yes, but you have `self.p` in the `__init__` and that should be changed.

Reply:
Oh yeah you are correct! will be changed